keyboards/handwired/dactyl_manuform/5x6_2_5/5x6_2_5.c

```diff
@@ -0,0 +1,2 @@
+#include "5x6_2_5.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Done.